package.json

```
@@ -37,7 +37,7 @@
   "dependencies": {
     "hastscript": "^5.0.0",
     "parse-entities": "^2.0.0",
-    "prismjs": "~1.20.0"
+    "prismjs": "^1.21.0"
```

```suggestion
    "prismjs": "~1.21.0"
```

The tilde is there for a reason. Prism sometimes breaks in minors, so we use tildes.